src/wui/mapdetails.cc

```diff
@@ -46,18 +49,41 @@ MapDetails::MapDetails(
                  "",
                  UI::Align::kLeft,
                  UI::MultilineTextarea::ScrollMode::kNoScrolling),
-     descr_(&main_box_, 0, 0, UI::Scrollbar::kSize, 0, style, ""),
-     suggested_teams_box_(new UI::SuggestedTeamsBox(this, 0, 0, UI::Box::Vertical, padding_, 0)) {
+     descr_(&descr_box_,
+            0,
+            0,
+            UI::Scrollbar::kSize,
+            0,
+            style,
+            "",
+            UI::Align::kLeft,
+            UI::MultilineTextarea::ScrollMode::kNoScrolling),
+     minimap_icon_(&descr_box_, 0, 0, 0, 0, nullptr),
+     suggested_teams_box_(new UI::SuggestedTeamsBox(this, 0, 0, UI::Box::Vertical, padding_, 0)),
+     egbase_(nullptr) {
+
+	minimap_icon_.set_frame(g_style_manager->minimap_icon_frame());
+	descr_.set_handle_mouse(false);
+	descr_box_.set_force_scrolling(true);
+
+	descr_box_.add(&descr_, UI::Box::Resizing::kAlign, UI::Align::kLeft);
```

Comment:
Try out the other `UI::Box::Resizing::` options, this can let the `descr_` fill the remaining space in the box automatically. You can then red rid of some of the manual layouting code below.

Reply:
If I try other options (kExpandBoth would be reasonable), the height is not calculated correctly and the box does not fit. Looks like a bug, as the same behavior can be observed on the campaign screen.
![Bildschirmfoto vom 2020-09-03 12-10-50](https://user-images.githubusercontent.com/23122583/92102428-e5136e80-edde-11ea-8956-c1653b694c95.png)